Reword backporting eligibility (#6356)
jenkins-infra/jenkins.io: content/download/lts/index.adoc

## Backporting Process

Any user can propose that an issue is backported to LTS by labeling with link:https://issues.jenkins.io/secure/IssueNavigator.jspa?reset=true&jqlQuery=labels+%3D+lts-candidate[lts-candidate].

> For an issue to be eligible for backporting it should be a small, safe-looking, uncontroversial fix for an important bug with no change to API surface. +

Reviewer: Is the line break intentional?  The trailing '+' forces a new line but does not create a new paragraph.  I'm fine with it if it is intentional, but found it surprising to see the line break.  It seemed to me that it should either be a paragraph break or be included in the flow of the existing paragraph
```suggestion
For an issue to be eligible for backporting it should be a small, safe-looking, uncontroversial fix for an important bug with no change to API surface.
```

> Backporting candidates have to be delivered in a weekly release to be tested by the community before shipping in LTS lines.

Reviewer: Thanks.  I think the "should be" phrasing from the previous sentence should be applied here as well, since there may be an exceptional case where we would consider a backport without it being in a weekly release.
```suggestion
Backporting candidates should be delivered in a weekly release before shipping in an LTS release.
```
